scripts/run_mypy_checks_test.py

```diff
@@ -120,32 +138,36 @@ def test_all_files_and_folders_in_not_fully_covered_files_exist(
             self.assertTrue(
                 os.path.exists(path), msg='"%s" does not exist' % path)
 
-    def test_install_third_party_libraries_with_skip_install_as_true(self):
+    def test_install_third_party_libraries_with_skip_install_as_true(
+        self
+    ) -> None:
         run_mypy_checks.install_third_party_libraries(True)
 
-    def test_install_third_party_libraries_with_skip_install_as_false(self):
+    def test_install_third_party_libraries_with_skip_install_as_false(
+        self
+    ) -> None:
         with self.install_swap:
             run_mypy_checks.install_third_party_libraries(False)
 
-    def test_get_mypy_cmd_without_files(self):
+    def test_get_mypy_cmd_without_files(self) -> None:
         expected_cmd = [
             self.mypy_cmd_path, '--exclude', 'file1.py|file2.py|dir1/|dir2/',
             '--config-file', './mypy.ini', '.'
         ]
         with self.files_swap:
             with self.directories_swap:
                 cmd = run_mypy_checks.get_mypy_cmd(
-                    None, self.mypy_cmd_path, False)
+                    None, self.mypy_cmd_path, False) # type: ignore[arg-type]
```

Comment:
Should we add TODO here to remove this later?

Reply:
Yeah, sure. I think we don't have any issue like that. please correct me if I am wrong so that I can create one.

Reply:
Can we use this issue https://github.com/oppia/oppia/issues/13528?

Reply:
sure